src/rgw/rgw_reshard.cc

```diff
@@ -378,15 +383,16 @@ int RGWBucketReshard::cancel()
 
 class BucketInfoReshardUpdate
 {
+  const DoutPrefixProvider *dpp;
```

Comment:
@cbodley ditto.

Reply:
@cbodley bump on this one. BucketInfoReshardUpdate isn't a CR or a thread.

Reply:
yeah this class just lives on the stack for a short time, that should be fine. it's also going away in reshard work